packages/apps/graph-client/src/constants/search.ts

```diff
@@ -0,0 +1,38 @@
+export enum SearchType {
+  GasEstimation = 'gasEstimation',
+  Account = 'account',
+  Block = 'block',
+  Transactions = 'request-key',
+  Event = 'event',
+}
+
+export const searchTypeLabels: Record<string, string> = {
+  'request-key': 'Request Key',
+  account: 'Account',
+  event: 'Event Name',
+  block: 'Block Hash',
+  gasEstimation: 'Cmd',
+};
+export const secondSearchTypeLabels: Record<string, string> = {
+  account: 'Module',
+  gasEstimation: 'Hash',
```

Comment:
What kind of Hash is this? Block?

Reply:
No it's the transaction hash, which when submitted will be the request key

Reply:
![image](https://github.com/kadena-community/kadena.js/assets/112877112/2fcb0e2f-4902-4661-ac0f-95229f0960d7)